uktrade/data-hub-frontend
# Add delete pipeline item form

## Description of change

Add ability to delete an archived pipeline item. This is currently hidden behind a url.

## Test instructions

Add a company to your pipeline, edit the pipeline item and then change the url from `/edit` to `/archive`, enter a reason and archive the item. Now click edit again and change `/edit` to `/delete` then click the delete button. The item should now be removed from your pipeline.

## Screenshots

![delete_form_1](https://user-images.githubusercontent.com/1481883/85136916-498b4c80-b238-11ea-875a-9abb4ff80fa3.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
